modules/nw-gcp-installing-global-access-configuration.adoc

```diff
@@ -5,7 +5,7 @@
 
 [id="nw-gcp-global-access-configuration_{context}"]
 = Create an Ingress Controller with global access on GCP
-You can create an Ingress Controller with global access on a new GCP cluster when your. Global access is only available to Ingress Controllers using internal load balancers.
+You can create an Ingress Controller that has global access to the GCP cluster. Global access is only available to Ingress Controllers using internal load balancers.
```

Comment:
"to the" or "to a"?

Reply:
Also, "using internal," "by using internal," or "that use internal?"

Reply:
@yunjiang29 Can you tell me which of these options it would be?

Reply:
@tsze-redhat PTAL, thanks